### PR TITLE
Fixes travis script to fail fast

### PR DESCRIPTION
**Pull request by vijetmahabaleshwar-okta**
Resolves https://oktainc.atlassian.net/browse/OKTA-229896

### CODE REVIEW

File: scripts/travis.sh [hunk 1]
```diff
@@ -3,9 +3,11 @@
 # if this build was triggered as cron job, run e2e tests on iOS
 if [ "${TRAVIS_EVENT_TYPE}" = "cron" ] ; then
     export SAUCE_PLATFORM_NAME="iOS";
+    set -e
     yarn build:release
     yarn test:e2e
 else
+    set -e
```
Comment (haishengwu-okta) [inline scripts/travis.sh]: curious what's `set -e`?

Comment (vijetmahabaleshwar-okta) [reply scripts/travis.sh]: It’s a `set error` command in shell script, which makes the script to exit as soon as the first thing fails.
In our case, the `yarn lint` failed, but it still continued to next line and finally `yarn test:e2e` passed. Which marked the build as success.
For us, we need to exit as soon as the first command fails.

Comment (vijetmahabaleshwar-okta) [reply scripts/travis.sh]: https://dougrichardson.org/2018/08/03/fail-fast-bash-scripting.html

Comment (vijetmahabaleshwar-okta) [reply scripts/travis.sh]: Also @haishengwu-okta, speeding up the build is definitely something we can look into, but I think that should be taken up as a separate task. For now, we should get the failing tests pass and get this PR merged?

Comment (mauriciocastillosilva-okta) [reply scripts/travis.sh]: I thought we needed to report at the end, so we can see all the things that failed (lint, uint test, etc)

Comment (vijetmahabaleshwar-okta) [reply scripts/travis.sh]: @mauriciocastillosilva-okta: We can do that as well. Need to extract the exit code after each command. And then report at the end. Do you think that’s better? There are pros and cons for both approaches

**Pull request merged**